app/js/controllers/angel.js

```diff
@@ -0,0 +1,253 @@
+'use strict';
+/* global angular, L */
+/* jshint unused:false */
+
+var controllersModule = require('./_index');
+var result;
+var startupslocation=[];
+var marker=[];
+var plotted=0;
+/**
+ * @ngInject
+ */
+
+ controllersModule.controller('AngelCtrl', ['$rootScope','$location','$http','$scope','AppSettings','MapPopUpTemplateService', function($rootScope,$location,$http,$scope,AppSettings,MapPopUpTemplateService) {
+  
+
+
+  var totalpage=0;
+  var currentpage=0;
+  $scope.startups=[];
+  var startupslayer = new L.LayerGroup();
+  var overlays = {
+            "Startups" : startupslayer 
+        };
+  var grayscale=L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+            maxZoom: 18,
+            attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, ' +
+        '<a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, ' +
+        'Imagery © <a href="http://mapbox.com">Mapbox</a>' +
+                'Imagery © <a href="http://mapbox.com">Mapbox</a>',
+            id: 'examples.map-20v6611k'
+        });
+        var basemapObj = {
+            "First Basemap": grayscale
+        };
+      
+      var map2 =L.map('map2',{layers:[grayscale,startupslayer]}).setView([20,0], 2);
+      
+      var code = function () {
+      var query_string = {};
+      var query = window.location.search.substring(1);
+      var vars = query.split("&");
+      for (var i=0;i<vars.length;i++) {
+        var pair = vars[i].split("=");
+        // If first entry with this name
+        if (typeof query_string[pair[0]] === "undefined") {
+        query_string[pair[0]] = decodeURIComponent(pair[1]);
+        // If second entry with this name
+      } else if (typeof query_string[pair[0]] === "string") {
+          var arr = [ query_string[pair[0]],decodeURIComponent(pair[1]) ];
+           query_string[pair[0]] = arr;
+        // If third or later entry with this name
+      } else {
+        query_string[pair[0]].push(decodeURIComponent(pair[1]));
+        }
+      } 
+    return query_string;
+    }();
+
+    $http.jsonp('https://angel.co/api/oauth/token?callback=JSON_CALLBACK', { params:
+    {
+      code : code,
+      grant_type:'authorization_code'
+    }}).then(function(response) {
+      //$location.path('/startups');
+  }, function(response) {
+
+      //$location.path('/startups');
+    // called asynchronously if an error occurs
+    // or server returns response with an error status.
+  });
+  $scope.curr_page=0;
+  $scope.lastpage;
+ 
+
+    var url="https://api.angel.co/1/startups?filter=raising&access_token=8cd27d851f1e4b69e27a29cc9606822c118586d4c8b168b5&callback=JSON_CALLBACK";
+    console.log(url);
+    $http.jsonp(url).
+    then(function(response) {
+      $scope.curr_page=response.data.page;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'curr_page' is not in camel case.](https://www.codacy.com/app/sudheesh1995/loklak_webclient/file/2550181458/issues/source?bid=3288768&fileBranchId=3288808#l80)